docs/user/get_started.md

```diff
@@ -0,0 +1,253 @@
+# Quick Start with garage
+
+Table of Content
+- [What is garage?](#what-is-garage)
+- [Why garage?](#why-garage)
+- [Kick Start garage](#kick-start-garage)
+- [Open Source Support](#open-source-support)
+- [Resources](#resources)
+
+
+## What is garage?
+
+garage is a reinforcement learning (RL) toolkit for developing and evaluating algorithms. The garage library also provides a collection of state-of-the-art implementations of RL algorithms.
```

Comment:
```suggestion
Garage is a reinforcement learning (RL) toolkit for developing and evaluating algorithms. The garage library also provides a collection of state-of-the-art implementations of RL algorithms.
```

Reply:
garage should be small letter instead of capital.